### ISSUE TITLE
Show 404 page for /{bad-url}

### ISSUE THREAD

**Issue by nikhilwoodruff** [PolicyEngine/policyengine-app#254]
Instead of routing to the default country.

**Comment by PGrad** [comment]
On it

**Comment by PGrad** [comment]
@nikhilwoodruff I've examined this problem in more detail, and I've come to the conclusion that redirecting to a 404 page for a non-existant country id is not intuitive for users, for a few reasons.

Even when the user sees the 404 page, they will see a header and footer that direct them back to some valid country page, say the UK page. The alternative is to not show the header and footer, which would trap them on the 404 page. To me, if the user is going to have to manually switch back to the UK page anyway, then they should be directed to that page automatically.

Perhaps instead of a 404 page, what you really want to do is provide a feedback form. Let's say the user wants to go to `policyengine.org/au` for Australia, but it's not available. We can provide a form for them to request an Australian page so we know there is a demand from users. Once they fill out the form, they have the option to check out an existing country page.

To recap, I see 2 solutions:
1. continue to automatically redirect to the UK page for non-existant country codes, or
2. let users provide feedback that they want to see a page for their country.

**Comment by PGrad** [comment]
Maybe this doesn't need a solution right now, I can focus on other items. In the meantime, I have another PR that adds onto my previous one by adding tests: https://github.com/PolicyEngine/policyengine-app/pull/255.

**Comment by nikhilwoodruff** [comment]
@PGrad - thanks, this is persuasive. OK, so yes of the two solutions I think the first is probably best.

**Comment by PGrad** [comment]
@nikhilwoodruff So is this issue safe to close?

**Comment by nikhilwoodruff** [comment]
Yep- closing now. Thanks for your help here!
